frontend/src/static/js/v_zoom.js

```diff
@@ -6,8 +11,22 @@ window.vZoom = {
       filterTimeFrame: window.hashParams.timeframe,
       showAllCommitMessageBody: true,
       expandedCommitMessagesCount: this.totalCommitMessageBodyCount,
+      commitsSortType: 'time',
+      toReverseSortedCommits: true,
+      sortingFunction: (a, b) => -1 * window.comparator(commitSortDict.time)(a, b),
```

Comment:
Why not making `sortingFunction` a computed value? So you do no need to watch `commitsSortType` and `toReverseSortedCommits` anymore?